### PR TITLE
Add scroll-to-top component

### PR DESCRIPTION
**Pull request by mativanarquero**
- [x] Add back to top button and component
    - Set btn color to #006bf8 to match 'Learn more' btn component
    - Only show btn if scrolled more than 300 px
    - Dynamic formatting across mobile resolutions

backtotop1
backtotop_r

### CODE REVIEW

**Review by harveyjavier — APPROVED** [review]
Elder scrolls!

File: src/pages/Home/ScrollButton.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  
+  return (
+    <Button>
+     <FaArrowCircleUp onClick={scrollToTop} 
+     style={{display: visible ? 'inline' : 'none'}} />
+    </Button>
+  );
+}
+  
+export default ScrollButton;
```
Comment (kbaldo) [inline src/pages/Home/ScrollButton.tsx:36]: Please add newline character here

**Review by swengr-janan — APPROVED** [review]
Scrolled!

**Review by kbaldo — COMMENTED** [review]
Oh. Missing newline character here haha

File: src/pages/Home/ScrollButton.tsx (hunk 1 above)
Comment (swengr-janan) [reply src/pages/Home/ScrollButton.tsx:36]: aw hahaha sorry

Comment (mativanarquero) [reply src/pages/Home/ScrollButton.tsx:36]: Done Kevs. Thank you!

**Pull request merged**